modules/calcite/src/test/java/org/apache/ignite/testsuites/ExecutionTestSuite.java

```diff
@@ -53,6 +54,7 @@
     IntersectExecutionTest.class,
     RuntimeSortedIndexTest.class,
     LimitExecutionTest.class,
+    LogicalRelImplementorTest.class,
```

Comment:
Execution test suite is supposed to verify actual execution of particular node or group of nodes. Let's add this test to the IgniteCalciteTestSuite.

Reply:
Fixed